docs/semgrep-app/findings.md

```diff
@@ -39,48 +63,54 @@ The Semgrep App Finding page displays findings across all projects connected to
 | 30-day findings retention | Community (Free) |
 | 1-year findings retention | Team/Enterprise |
 
-## Viewing data in the Findings page
+## Viewing data on the Findings page
 
 ### Navigating to the Findings page
 
 1. Sign in to Semgrep App.
 2. The **Findings** page can be found on the left sidebar.
 
-### Understanding the Findings data
+### Understanding the Findings page
 
 The Findings page displays the following fields:
 
-| Field       | Description |
+| Field       | Description  |
 | ----------  | ------------ |
-| **Severity**    | The impact of a finding. Possible values: Low, Medium, High, and Critical. |
-| **Finding**     | The file, line, and branch of the pattern match in the code. Clicking on this field brings you to the exact location in the codebase. |
-| **Project**     | The name of the project codebase where the finding was found. Clicking on this field brings you to the project’s URL. |
-| **Rule**        | The name of the rule matched with the code. Clicking on this field brings you to the rule’s entry in the [rule registry](https://semgrep.dev/r). |
-| **Introduced**  | The time when this finding was first found. Clicking on the header row sorts the table by latest or earliest findings. |
-| **Status**      | Refers to a finding’s state. |
+| **Severity**    | The impact of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High </li><li>Critical</li></ul> |
+| **Finding**     | The file, line, and branch of the match in the code. Clicking this field brings you to the exact location in the codebase. |
+| **Project**     | The name of the repository where the finding was found. Clicking this field brings you to the project’s URL. |
+| **Rule**        | The name of the rule that  matched with the code. Clicking this field brings you to the rule’s entry in the [Rule Registry](https://semgrep.dev/r). |
```

Comment:
Regarding the link to Registry, what about for rules that aren’t in the Registry, like private rules or rules hosted somewhere else?